applications/tari_console_wallet/src/ui/state/mod.rs

```diff
@@ -25,4 +25,31 @@ mod debouncer;
 mod tasks;
 mod wallet_event_monitor;
 
+use serde::{Deserialize, Serialize};
+use tari_common_types::serde_with;
+
 pub use self::app_state::*;
+
+// ----------------------------------------------------------------------------
+// TODO: re-implement in a clean way
```

Comment:
"Clean way" is subjective. Maybe expand on what needs to be done here. I don't think the comment is needed. This is suitable for now

Reply:
"Clean way" was a poor word choice, cleanliness here is more like finding a logical place to put these transient types to